src/style.css

```diff
@@ -0,0 +1,102 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  font-size: 11px;
+  line-height: 13px;
```

Comment:
[sort](https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/) rules